.eslintrc-base

```diff
@@ -1,7 +1,2 @@
 ---
-"plugins":
-  - "filenames"
-"rules":
-  "filenames/filenames": [2, "^[a-z\\-\\.]+$"] # dash-cased filenames.
-"globals":
-  "expect": false
\ No newline at end of file
+# Any base overrides can go here.
```

Comment:
If you follow the chain of `extends`, the `filenames` settings are now a part of the Walmart config we're already extending.